filters/openpolicyagent/openpolicyagent.go

```diff
@@ -37,6 +37,7 @@ const (
 	defaultReuseDuration       = 30 * time.Second
 	defaultShutdownGracePeriod = 30 * time.Second
 	DefaultCleanIdlePeriod     = 10 * time.Second
+	ResponseHeadersKey         = "open-policy-agent:decision-response-headers"
```

Comment:
Why do we need this exported?

Reply:
It doesn't need to be. Moved the constant to the class it is used.